conference.js

```diff
@@ -639,7 +649,10 @@ export default {
         // overshadowed by the overlay.
         tryCreateLocalTracks.then(tracks => {
             APP.store.dispatch(toggleSlowGUMOverlay(false));
-            APP.store.dispatch(mediaPermissionPromptVisibilityChanged(false));
+            if (tracks.length >= 2) {
+                // granted audio and video
+                APP.store.dispatch(mediaPermissionPromptVisibilityChanged(false));
```

Comment:
could this change explain why the prompt didn't disappear in today's meeting (don't remember who ran into this)?

Reply:
Could be but I believe the issue comes from the app itself where I am conditionally hiding/showing the screen. I will look into it today.